Maui/MLTrainer/MLTrainer.csproj

```diff
@@ -53,6 +53,7 @@
 	<ItemGroup>
 		<PackageReference Include="CommunityToolkit.Mvvm" />
 		<PackageReference Include="CsvHelper" />
+    <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.0-rc.2.9511" />
```

Comment:
added to address this:

warning MA002: Starting with .NET 8, setting  <UseMaui>true</UseMaui>  does not automatically include NuGet package references in your project.  Update your project by including this item:  <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.0-rc.2.9511" />.  You can skip this warning by setting  <SkipValidateMauiImplicitPackageReferences>true</SkipValidateMauiImplicitPackageReferences>  in your project file.